base_site/index.html

```diff
@@ -1,96 +1,222 @@
+          </div>
+        </div>
+      </header>
+      <section class="section_products-list">
+        <div class="padding-global">
+          <div class="container">
+            <div class="padding-section-small">
+              <div class="product_list-wrap">
+                <div class="w-layout-grid product-list_component is-first">
+                  <div class="product-list_content">
+                    <div class="product_logo-wrap"><img src="images/nexus-wordmark.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <div class="spacer-small"></div>
+                    <p class="product_description">Cloud platform access and manage our full-stack</p>
+                    <div class="spacer-medium"></div>
+                    <a href="https://docs.quantinuum.com/nexus/index.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with Nexus</p>
+                      <p class="product_description is-list">Our platform for accessing and managing the highest performance full-stack</p>
+                    </a>
+                    <div class="spacer-small"></div>
+                    <a href="https://docs.quantinuum.com/nexus/user_guide/sign_up.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Tutorials and Documentation</p>
+                      <p class="product_description is-list">Find the latest technical documentation and  additional resources</p>
+                    </a>
+                  </div>
+                  <div id="w-node-c6173a0f-f663-21c6-62d6-04d227112a72-c2a09c1d" class="product-list_image-wrapper is-nexus"><img src="images/nexus-sized.png" loading="lazy" alt="" class="product-list_image"></div>
+                </div>
+                <div class="w-layout-grid product-list_component">
+                  <div class="product-list_content">
+                    <div class="product_logo-wrap"><img src="images/h-series.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <div class="spacer-small"></div>
+                    <p class="product_description">The world’s highest-performing  quantum hardware</p>
+                    <div class="spacer-medium"></div>
+                    <a href="https://docs.quantinuum.com/h-series/index.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with H-Series</p>
+                      <p class="product_description is-list">Explore how to use the industry&#x27;s leading  quantum processors</p>
+                    </a>
+                    <div class="spacer-small"></div>
+                    <a href="https://docs.quantinuum.com/h-series/user_guide/hardware_user_guide/access.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Tutorials and Documentation</p>
```

Comment:
```suggestion
                    <a href="https://docs.quantinuum.com/h-series/user_guide/hardware_user_guide/access.html" class="product_link-wrap w-inline-block">
                      <p class="product_link-text">Get Started with H-Series</p>
                      <p class="product_description is-list">Explore how to use the industry&#x27;s leading  quantum processors</p>
                    </a>
                    <div class="spacer-small"></div>
                    <a href="https://docs.quantinuum.com/h-series/index.html" class="product_link-wrap w-inline-block">
                      <p class="product_link-text">Tutorials and Documentation</p>
```

Swap around the "Get started" and "Tutorials and Documentation" links.